src/classification.html

```diff
@@ -44,118 +45,191 @@
-		</div><!--hero-unit-->
+		</div>
+	</header>
 
-		<div  class="container" id="tosdr-content" role="main">
-				
-        			<p>
-        				Terms of service are reviewed by contributors and divided into small points that we can discuss, <a href="topics.html">compare</a> and ultimately assign a score with a badge: <br /><span class="badge badge-success"><i class="icon-thumbs-up icon-white">+</i> Good</span>, <span class="badge badge-warning"><i class="icon-thumbs-down icon-white">-</i> Bad</span>, <span class="badge badge-important"><i class="icon-remove icon-white">×</i> Blocker</span>, or <span class="badge"><i class="icon-arrow-right icon-white">→</i> Neutral</span>.
-        			</p>
-        			
-        			<p>
-        				Once a service has enough badges to assess the fairness of their terms for users, a class is assigned automatically by pondering the average scores.
-        			</p>
-				<p>
-					<span class="label A" style="vertical-align:middle">Class A</span> are the best terms of services: they treat you fairly, respect your rights and will not abuse your data.
-				</p>
-				<p><span class="label B">Class B</span>&nbsp; The terms of services are fair towards the user but they  could be improved.</p>
-				<p><span class="label C">Class C</span>&nbsp; The terms of service are okay but some issues need your consideration.</p>
-				<p><span class="label D">Class D</span>&nbsp; The terms of service are very uneven or there are some important issues that need your attention.</p>
-				<p><span class="label E">Class E</span>&nbsp; The terms of service raise very serious concerns.</p>
-				<p><span class="label false">No Class Yet</span>&nbsp; We haven't sufficiently reviewed the terms yet.
+	<main id="main-content"><!--Main Body.-->
+		<article> <!--TODO Should we add a background? (style="background: url('img/tosdr-logo-128-low-opacity.png') no-repeat center;")-->
+			<section id="process">
+				<div class="container" id="tosdr-content" role="main">
+					<h2><strong>How we review</strong></h2>
+					<p>
+						When a new service is added to the database, its Terms are reviewed by <ins>contributors</ins>, who divide the pertinent documents (<strong>ToS</strong>, <strong>Privacy Policy</strong>, <strong>Cookie Policies</strong>, etc.) into small quotes ("<ins>points</ins>"), each referring to a specific <a href="https://edit.tosdr.org/cases" target="_blank"><strong><ins>Case</ins></strong></a> which will assign one of the following badges to the point:
+					</p>
+					<div id="ratings">
+						<span class="badge badge-success"><i class="icon-thumbs-up icon-white">+</i> Good&nbsp;</span>
+						<br>
+						<span class="badge badge-warning"><i class="icon-thumbs-down icon-white">-</i> Bad&nbsp;</span>
+						<br>
+						<span class="badge badge-important"><i class="icon-remove icon-white">×</i> Blocker&nbsp;</span>
+						<br>
+						<span class="badge"><i class="icon-arrow-right icon-white">→</i> Neutral&nbsp;</span>
+					</div>
```

Comment:
Still missing whitespaces, non breaking spaces wont work here.
```suggestion
					<div id="ratings">
						<span class="badge badge-success"><i class="icon-thumbs-up icon-white">+</i> Good </span>
						<br>
						<span class="badge badge-warning"><i class="icon-thumbs-down icon-white">-</i> Bad </span>
						<br>
						<span class="badge badge-important"><i class="icon-remove icon-white">×</i> Blocker </span>
						<br>
						<span class="badge"><i class="icon-arrow-right icon-white">→</i> Neutral </span>
					</div>
```